P/PARMETIS/build_tarballs.jl

```diff
@@ -63,12 +85,15 @@ platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), plat
 
 # The products that we will ensure are always built
 products = [
-    LibraryProduct("libparmetis", :libparmetis)
+    LibraryProduct("libparmetis", :libparmetis),
+    LibraryProduct("libparmetis_Int32_Real64", :libparmetis_Int32_Real64),
+    LibraryProduct("libparmetis_Int64_Real32", :libparmetis_Int64_Real32),
+    LibraryProduct("libparmetis_Int64_Real64", :libparmetis_Int64_Real64)
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("METIS_jll"),
+    Dependency("METIS_jll", v"5.1.2"),
```

Comment:
If a specific version is important for building, probably it's also important to explicitly set compat:
```suggestion
    Dependency("METIS_jll"; compat="5.1.2"),
```